### ISSUE TITLE
Tensorboard event files are very large

### ISSUE THREAD

**Issue by phiresky** [IndicoDataSolutions/finetune#184]
**Describe the bug**
Tensorboard event files are 800MB+ per run

**Minimum Reproducible Example**

```python
import finetune

x = ["foo", "bar", "baz"]
y = [1, 0, 0]

classifier = finetune.Classifier(tensorboard_folder="./tb_test")

classifier.fit(x, y)
```

I'm not sure what exactly is contained in the log files - but this is pretty annoying because it means that after ~30 runs, tensorboard is taking 40+ Gigabyte of RAM for me. Looks like for some reason the graph definition is huge (reading the file with `tf.train.summary_iterator('./events.out....')`)

**Comment by phiresky** [comment]
might be caused by this? https://github.com/tensorflow/tensorflow/issues/1444

**Comment by madisonmay** [comment]
Yeah, I've noticed this too but haven't yet prioritized a solution.  Thanks for getting this ticket into github and doing some initial sleuthing!  40+ GB of RAM is definitely absurd -- I haven't experienced anything quite that bad before.  

We initially thought this was simply due to gradient summaries but that option is now off by default so it's not immediately clear what the root cause might be.  Will have to do some sleuthing of my own.

**Comment by benleetownsend** [comment]
This is caused by the issue referenced above, I have proposed a fix in #191 that allows us to keep the graph defs for viewing in tensorboard but removes the mass of constants. The resulting summaries are now much more compact.
